src/app/stacktrace_win.h

```diff
@@ -73,7 +73,7 @@ void straceWin::loadHelpStackFrame(IMAGEHLP_STACK_FRAME& ihsf, const STACKFRAME6
 BOOL CALLBACK straceWin::EnumSymbolsCB(PSYMBOL_INFO symInfo, ULONG size, PVOID user)
 {
     Q_UNUSED(size)
-    QStringList* params = (QStringList*)user;
+    auto *params = static_cast<QStringList *>(user);
```

Comment:
You don't need to add `*` to `auto` if the right part clearly expresses that it is a pointer. This only improves readability in less obvious cases, for example.: `auto session = BitTorrent::Session::instance();`

Reply:
Changed.